src/python/gudhi/sklearn/cubical_persistence.py

```diff
@@ -98,7 +92,7 @@ def __transform(self, cells):
     def transform(self, X, Y=None):
         """Compute all the cubical complexes and their associated persistence diagrams.
 
-        :param X: List of cells filtration values (`numpy.reshape(X, newshape, order='C'` if `newshape` is set with a tuple of ints).
+        :param X: List of cells filtration values.
```

Comment:
Maybe we could be a bit clearer, though when I try I mostly end up repeating the type :-(
Filtration values of the top-dimensional cells or vertices for each complex?

Reply:
I took your proposal on [c0caed4](https://github.com/GUDHI/gudhi-devel/pull/856/commits/c0caed4de511df2695079eaf93b28bed860d59b7), thanks !